code/datums/dog_fashion.dm

```diff
@@ -10,6 +10,8 @@
 	// sprite that the associated item uses
 	var/icon_file
 	var/icon_state
+	var/alpha
+	var/color
```

Comment:
it works you know, i don't play around with my testing. but @Iamgoofball it works for the same reason icon_state works, this is a datum.